pyfar/dsp/fft.py

```diff
@@ -301,16 +404,21 @@ def _n_bins(n_samples):
     ---------
     n_samples : int
         Number of samples
+    complex : bool
+        Flag which indicates if the time data are real or complex-valued.
 
     Returns
     -------
     n_bins : int
         Resulting number of frequency bins
 
     """
-    if _is_odd(n_samples):
-        n_bins = (n_samples+1)/2
+    if complex:
+        n_bins = n_samples
     else:
-        n_bins = n_samples/2+1
+        if _is_odd(n_samples):
+            n_bins = (n_samples+1)/2
+        else:
+            n_bins = n_samples/2+1
```

Comment:
I think this can be replaced by a one-liner.
```suggestion
        n_bins = n_samples // 2 + 1
```
This might make `_is_odd` obsolete and would directly return an int